### DISCUSSION TITLE
setting up SO on Ubuntu 18.04 issue

### DISCUSSION THREAD

**Discussion by dbui1910** [Security-Onion-Solutions/securityonion#4921] [Unsupported Versions] [answered]
I am setting Security Onion on Ubuntu 18.04. I pick 2 NICs ( one using NAT, the other one is VMnet19). However, during the setup SO doesn't show up anything for me to assign the IP address for NIC. I also got the message saying: "en34 is unmanaged by Network Manager". Could anyone have any idea or experience ob this issue?
<img width="865" alt="unmanaged" src="https://user-images.githubusercontent.com/68297071/126846300-7cd58820-4f26-46c1-8200-6824269e36dd.png">

**Comment by dougburks** [comment, marked as answer]
> I am setting Security Onion on Ubuntu 18.04. I pick 2 NICs ( one using NAT, the other one is VMnet19). However, during the setup SO doesn't show up anything for me to assign the IP address for NIC.

Since you're doing a network installation, you should have already configured your IP address.

> I also got the message saying: "en34 is unmanaged by Network Manager".

Please try the recommendation at the end of the Installation page:
https://docs.securityonion.net/en/2.3/installation.html#installation-on-ubuntu-or-centos

**Comment by ghost** [comment]
> I am setting Security Onion on Ubuntu 18.04. I pick 2 NICs ( one using NAT, the other one is VMnet19). However, during the setup SO doesn't show up anything for me to assign the IP address for NIC.

I've run into both of your situations. As for this one, I struggled with this for a very long time when I first started using SO. I found out by accident that disabling the second/monitoring NIC (the one going to the SPAN port) during Ubuntu setup helps tremendously. If I don't do that, the SO installation will not configure the bond0 properly. At least, it wouldn't for me. Set the management NIC IP address during this step. I have attached the instructions I made for myself.

[Disable Monitoring NIC.docx](https://github.com/Security-Onion-Solutions/securityonion/files/6947278/Disable.Monitoring.NIC.docx)

>I also got the message saying: "en34 is unmanaged by Network Manager". Could anyone have any idea or experience ob this issue?

I ran into this last week when trying to upgrade my remote nodes without reinstalling Ubuntu. I got the exact same error message. I resolved it by stopping Network Manager prior to running the SO setup. I used `sudo stop network-manager`. The SO installation and reboot will start it up again.

Hope that helps. Good luck!
